DXRModules/DeusEx/Classes/DXRBingoCampaign.uc

```diff
@@ -129,6 +129,46 @@ function FirstEntry()
     }
 }
 
+function PostFirstEntry()
+{
+    switch (dxr.localURL) {
+        case "04_NYC_STREET":
+        case "04_NYC_HOTEL":
+        case "04_NYC_BatteryPark":
+            HandleSpecialPerson("AnnaNavarre", "AnnaNavarre_DeadM5");
+            break;
+        case "14_VANDENBERG_SUB":
+        case "14_OceanLab_Lab":
+        case "14_OceanLab_UC":
+            HandleSpecialPerson("WaltonSimons", "WaltonSimons_Dead");
+            break;
+    }
+}
+
+function HandleSpecialPerson(string bindname, string goal) {
+    local int oldSeed;
+    local ScriptedPawn pawn;
+    local PlayerDataItem data;
+
+    if (IsBoardGoal(goal)) return;
+
+    data = class'PlayerDataItem'.static.GiveItem(player());
+    oldSeed = SetGlobalSeed(dxr.dxInfo.MissionNumber @ dxr.flags.newgameplus_loops $ " HandleSpecialPerson " $ bindname);
+
+    if (chance_single(50)) {
+        foreach AllActors(class'ScriptedPawn', pawn) {
+            if (pawn.bindname == bindname) {
+                pawn.LeaveWorld();
```

Comment:
LeaveWorld won't work here because they're already out of world and then EnterWorld when it's time for them to appear (Anna and Walt don't appear until you've sent the signal/got the UC schematics)

just Destroy() them